garak/probes/visual_jailbreak.py

```diff
@@ -41,15 +42,14 @@ class FigStep(Probe):
 
     modality: dict = {"in": {"text", "image"}}
 
-    safebench_image_catalog = (
-        _config.transient.package_dir / "resources" / "safebench_filenames.txt"
-    )
+    safebench_image_catalog = data_path / "safebench_filenames.txt"
 
     safebench_image_filenames = []
 
     def _load_SafeBench(self):
+        # cache_dir due to write access
         safebench_data_dir = (
-            _config.transient.cache_dir / "resources" / "visual_jailbreak" / "SafeBench"
+            _config.transient.cache_dir / "data" / "visual_jailbreak" / "SafeBench"
```

Comment:
Here is a case where I think the cached file is probe specific and pathing reflects the intent.